pkg/config/log.go

```diff
@@ -350,6 +325,24 @@ func extractShortPathFromFullPath(fullPath string) string {
 	return slices[len(slices)-1]
 }
 
+func changeSeelogLevel(level string) error {
```

Comment:
Since this also changes the level of our own logger, shouldn't this be renamed to:
```suggestion
func changeLogLevel(level string) error {
```

Reply:
Will do.